rollup.config.js

```diff
@@ -1,27 +1,34 @@
+/* eslint-disable import/no-default-export */
+
 import babel from 'rollup-plugin-babel';
 import commonjs from 'rollup-plugin-commonjs';
 import resolve from 'rollup-plugin-node-resolve';
 
 const extensions = ['.ts', '.tsx'];
 
-module.exports = {
-    input: ['src/index.ts', 'src/component/ConsentManagementPlatform.tsx'],
-    output: [
-        {
-            dir: 'lib',
-            format: 'cjs',
-        },
-        {
-            dir: 'lib',
-            format: 'cjs',
-        },
-    ],
+export default () => ({
+    input: {
+        index: 'src/index.ts',
+        ConsentManagementPlatform:
+            'src/component/ConsentManagementPlatform.tsx',
+    },
+    output: {
+        dir: 'lib',
+        format: 'cjs',
+    },
     external: [
         'react',
         '@emotion/core',
         '@guardian/src-button',
         '@guardian/src-foundations',
         '@guardian/src-svgs',
+        '@guardian/src-foundations/mq',
```

Comment:
By setting these to be external don't we need to set them as peerDependencies?

Reply:
def, but `@guardian/src-foundations` is defined already and would include the modules that we use from within it
 https://github.com/guardian/consent-management-platform/blob/f0200337468091868eaed1d9885ff305c1f9056a/package.json#L36-L43